lib/prolongator_mma.in.cu

```diff
@@ -0,0 +1,244 @@
+#include <color_spinor_field.h>
+#include <multigrid.h>
+#include <tunable_nd.h>
+#include <expand_list.hpp>
+
+#ifdef QUDA_MMA_AVAILABLE
+#include <device.hpp>
+#include <kernels/prolongator_mma.cuh>
+#include <mma_tensor_op/smma_m16n8k8_sm70.cuh>
+#endif
+
+namespace quda
+{
+
+#ifdef QUDA_MMA_AVAILABLE
+
+  template <typename Float, typename vFloat, int fineSpin, int fineColor, int coarseSpin, int coarseColor, int nVec>
+  class ProlongateLaunchMma : public TunableKernel
+  {
+
+    ColorSpinorField &out;
+    const ColorSpinorField &in;
+    const ColorSpinorField &V;
+    const int *fine_to_coarse;
+    int parity;
+    QudaFieldLocation location;
+
+    // using mma_t = typename mma::smma_dispatch<Float>::type;
+    // using mma_t = simt::simt_t<float, 8, 4, 2, 2>;
+    // using mma_t = smma::smma_x_t<mma::half, 8, 1, 1>;
+    // using mma_t = hmma::hmma_t<16, 16, 4, mma::half, mma::half2>;
+#if (__COMPUTE_CAPABILITY__ >= 800)
+    using mma_t = hmma::hmma_tfloat32_t<4, 1, 1>;
+#else
+    using mma_t = hmma::hmma_x_t<16, 8, 8, mma::half, mma::half2>;
```

Comment:
Should this be float instead of half? I'm just not sure of the convention here

Reply:
It's either TF32 or FP16 here so I think it's right.